process/project_proposals.adoc

```diff
@@ -70,8 +70,13 @@ image::incubation-process.png[Incubation process]
 
 === Graduation process
 
-Graduation process follows the pattern of incubation, but the majority of DD should have happened at the incubation phase so it should be a less onerous process for graduation.
-
+. *Submit Graduation Proposal Template*
+   * Project fills out and submits the [graduation proposal template](graduation-proposal-template.md) in a pull request in the [cncf/toc GitHub repo](https://github.com/cncf/toc)
```

Comment:
Does the existing proposal doc need to be updated? For example, [the one for Helm notes it's incubating](https://github.com/cncf/toc/blob/master/proposals/helm.adoc). If so, what should be updated?

Reply:
Nope. Good question. You can leave that one in place for historical record. You'll create a new proposal which we will put under a directory called `proposals/gradution/` See #375 for the start of that reorganization.